common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/SwitchBrowserActivity.kt

```diff
@@ -147,7 +147,6 @@ class SwitchBrowserActivity : FragmentActivity() {
             Logger.warn(methodTag, "CustomTabsService is NOT supported")
             browserIntent = Intent(Intent.ACTION_VIEW)
         }
-        browserIntent.setPackage(browserPackageName)
```

Comment:
I’m wondering if we should set the intent package in the else condition when CCT isn’t supported, so that the desired browser is used

Reply:
Addressed